CONTRIBUTING.md

```diff
@@ -301,3 +305,28 @@ Then in the `pybricks-micropython` directory:
     yarn build:debug
 
 [wasm]: https://webassembly.org/
+
+Build and deploy firmware
+-------------------------
+
+### Build the firmware
+
+Pick your Hub from the `bricks` sub-directory you want to compile.
+
+    poetry shell
+    make mpy-cross -j8
+    make -C bricks/primehub -j8
+       
+### Deploy the firmware to a hub
+
+First, follow the guide to turn your Hub into DFU mode:
```

Comment:
For non-USB hubs, they also need to be in bootloader mode, but technically it is not "DFU" mode since they don't have USB.

Reply:
OK, will re-phrase that.

Reply:
Done.